test/condition-travis-test.js

```diff
@@ -259,6 +259,31 @@ test.serial('Calls travis-run-once with pro parameter determined by github call
 
   t.falsy(result);
   t.true(travisDeployOnce.calledOnce);
-  t.deepEqual(travisDeployOnce.firstCall.args[0], {travisOpts: {pro}});
+  t.deepEqual(travisDeployOnce.firstCall.args[0], {travisOpts: {pro, enterprise: undefined}});
+  t.true(github.isDone());
+});
+
+test.serial('Calls travis-run-once with enterprise parameter if provided', async t => {
```

Comment:
The test title should describe what's expected. `Calls travis-run-once with enterprise parameter if provided` => `Calls travis-run-once with enterprise parameter`.